utils/scap-as-rpm.8

```diff
@@ -1,6 +1,6 @@
 .TH scap-as-rpm "8" "November 2013" "scap-as-rpm" "System Administration Utilities"
 .SH NAME
-scap-as-rpm \- manual page for scap-as-rpm
+scap-as-rpm \- takes given SCAP input and create an RPM package
```

Comment:
```suggestion
scap-as-rpm \- takes given SCAP input(s) and packs them in an RPM package 
```

Reply:
Applied the suggestion, except for the end-of-line whitespace